### ISSUE TITLE
Version 2.4.3, cloud foundry actuators are not working 

### ISSUE THREAD

**Issue by VenkateshSrini** [SteeltoeOSS/Steeltoe#327]
## Describe the bug
I'm using SteelToe with Pivotal cloud foundry Application Services. I use .NET core and host in cflinux3 stack. I have added all the necessary packages including 'SteelToe.Management.CloudFoundryCore` . I have added `AddCloudFoundry()` `AddCloudFoundryActuators` and `UseCloudFoundryHosting` in `CreateHostbuilder` in program.cs

I have added `services.AddCloudFondryActuators(Configuration)` in `ConfigureService` Method and `app.UseCloudFoundryActuators() ; app.UseCloudFoundrySecurity(); in Configure` method. 

I use a base path 'MyApp' for my application and hence my management settings looks as below in PCF environment 
```
"management": {
        "endpoints": {
           "path": "/myApp/cloudfoundryapplication"
            "cloudfoundry": {
                "validateCertificates": false,
               "enabled": true
            }
        }
   
}
```
In spite of all these settings, the Steeltoe icon is not appearing for the App, No dashboard of health checks is not working. This was working with earlier version 2.1.2 and .NET core 2.1. Now I'm converting my app from .NET core 2.1 to 3.1 and upgraded my library to 2.4.3. It stopped working. 

## Steps to reproduce 
As stated  above

## Expected behavior
Steel Toe actuators would work in PCF
## Environment (please complete the following information):
 - Platform: [e.g. Cloud Foundry, Azure, etc.) : PCF
 - OS: [e.g. Windows, Linux, Mac OSX]: CFLINUX 3
 - .NET Version [e.g. .NET Core 2.1.0, .NET Framework 4.7.1, etc.] : .NET core 3.1
 - Steeltoe Version [e.g. 2.3.0]: 2.4.2
 - Any other library versions to note

**Comment by hananiel** [comment]
@VenkateshSrini running in PCF the actuators path always needs to be /cloudfoundryapplication. With the newer versions, this is automatically done for you when you use the overload to set the context to ActuatorAndCloudFoundry or just Cloudfoundry. In this case, the base path should not be set in settings.

**Comment by VenkateshSrini** [comment]
Could you please be more descriptive? I'm unable to understand when you say `use the overload to set the context to ActuatorAndCloudFoundry or just Cloudfoundry`. Can you please provide the command that needs to be used and if so where, In program.cs where we do the `AddCloundfoundry` or in the Startup.cs where we do the `AddActuators`

**Comment by VenkateshSrini** [comment]
@hananiel ,
Thanks it worked. But I wanted to put up the steps I did  so that it will be useful for other users also

1. we have to put MediaType version parameter and AddActuatorContext in both configureservices and configure method UseCloudfoundry method invocation.
2. We need to add AddCloudFoundry() in ConfigureAppConfiguration in web host builder.
3. If we are using the base path then managements:endpoints:path to be `/myapp/cloudfoundryapplication`, assuming my base path is configured as myapp.

With these changes I got it working
